common/src/main/res/values/strings.xml

```diff
@@ -298,11 +298,20 @@
     <string name="gestures_action_server_list">Server list</string>
     <string name="gestures_action_server_next">Next server</string>
     <string name="gestures_action_server_previous">Previous server</string>
+    <string name="gestures_category_servers">Servers</string>
     <string name="gestures_description">Set up gestures for quick actions whenever you are using the main Home Assistant UI.\n\nNote: if your device has gestures that work in any app, they may override Home Assistant gestures.</string>
     <string name="gestures_direction_down">Swipe down</string>
     <string name="gestures_direction_left">Swipe left</string>
     <string name="gestures_direction_right">Swipe right</string>
     <string name="gestures_direction_up">Swipe up</string>
+    <string name="gesture_name_swipe_down_two">Swipe down using two fingers</string>
+    <string name="gesture_name_swipe_down_three">Swipe down using three fingers</string>
+    <string name="gesture_name_swipe_left_two">Swipe left using two fingers</string>
+    <string name="gesture_name_swipe_left_three">Swipe left using three fingers</string>
+    <string name="gesture_name_swipe_right_two">Swipe right using two fingers</string>
+    <string name="gesture_name_swipe_right_three">Swipe right using three fingers</string>
+    <string name="gesture_name_swipe_up_two">Swipe up using two fingers</string>
+    <string name="gesture_name_swipe_up_three">Swipe up using three fingers</string>
```

Comment:
These strings are new because simply using `direction + " " + number of pointers` may not work for all languages.